### ISSUE TITLE
How to change the selected node programmatically

### ISSUE THREAD

**Issue by sagIoTPower** [josdejong/svelte-jsoneditor#227]
Could you provide a sample how to set the selection to a specific path?
I don't see a way/method to change the selected node programmatically.

**Comment by sagIoTPower** [comment]
Hi,

I explored how this could be achieved and ran a local test.
First I exported
`export function updateSelection(selection: ....)`
much like 
`export function scrollTo(path: Path)`
in the following svelte scripts: 
```
src/lib/components/modes/tablemode/TableMode.svelte
src/lib/components/modes/treemode/TreeMode.svelte
```
Other files I had to touch were:
```
src/lib/components/modes/JSONEditorRoot.svelte
src/lib/components/JSONEditor.svelte
```
Finally I could set the selection.

Would it be possible to expose in JSONEditor?
`function updateSelection(selection: ....)`

Thanks
